labels.md

```diff
@@ -0,0 +1,220 @@
+type/bug-fix                    #fc2929     type/bug fix                        #1d76db
+type/compatibility              #e99695     type/compatibility                  #1d76db
+type/duplicate                  #cccccc     status/closed dup                   #e6e6e6
+type/enhancement                #84b6eb     type/enhancement                    #1d76db
+type/invalid                    #e6e6e6     status/closed invalid               #e6e6e6
+type/investigation              #f46100     type/investigation                  #1d76db
+type/new-feature                #e2e876     * <type/enhancement>
+type/performance                #006b75     type/performance                    #1d76db
+type/regression                 #c41b2f     type/regression                     #1d76db
+type/suggestion                 #11b2c4     * <type/proposal>                     
+type/wontfix                    #95d3db     * <just close the issue>            
+v3.0                            #cb11f9     *                                   
+```
+
+## TiKV
+
+```
+C: Build                        #d1fad7     component/build                     #d1fad7
+C: Build-Time                   #d1fad7     component/build time                #d1fad7
+C: Copr                         #d1fad7     component/coprocessor               #d1fad7
+C: Doc                          #d1fad7     component/docs                      #d1fad7
+C: PD-Client                    #d1fad7     component/PD client                 #d1fad7
+C: Perf                         #d1fad7     component/perf                      #d1fad7
+C: Raft                         #d1fad7     component/raft                      #d1fad7
+C: RocksDB                      #d1fad7     component/RocksDB                   #d1fad7
+C: Server                       #d1fad7     component/server                    #d1fad7
+C: Storage                      #d1fad7     component/storage                   #d1fad7
+C: Test/Bench                   #d1fad7     component/test                      #d1fad7
+                                            component/bench                     #d1fad7
+C: TiKV-Client                  #d1fad7     component/TiKV client               #d1fad7
+C: TiKV-Ctl                     #d1fad7     component/TiKV ctl                  #d1fad7
+C: Titan                        #d1fad7     component/Titan                     #d1fad7
+C: Txn                          #d1fad7     component/transactions              #d1fad7
+C: Util                         #d1fad7     component/util                      #d1fad7
+C: gRPC                         #d1fad7     component/gRPC                      #d1fad7
+D: Easy                         #0e8a16     difficulty/easy                     #0e8a16
+D: Medium                       #f4b169     difficulty/medium                   #0e8a16
+D: Mentor                       #31c639     difficulty/mentor                   #0e8a16
```

Comment:
BTW I'm curious what is a "mentor" difficulty XD

Reply:
can we remove this label?